site/topics/exceptions/exceptions.rst

```diff
@@ -211,9 +211,9 @@ Why Care?
 * The trouble is, that depends on your situation
 * The point is, how can the programmer in 1991 know what you want to do with your situation today?
 
-# Exceptions are fantastic because
+* Ultimately, exceptions are fantastic because
```

Comment:
_fantastic_ is probably a little strongly worded. It is arguable that exceptions are side effects, a lot of people like to use constructs like `Either` or `Try`, similar to python's `Option` for nullability.